src/sql/statements.cpp

```diff
@@ -259,39 +266,39 @@ SelectStatement::~SelectStatement() {
   delete limit;
 
   // Delete each element in the select list.
-  if (selectList != nullptr) {
+  if (selectList) {
     for (Expr* expr : *selectList) {
       delete expr;
     }
     delete selectList;
   }
 
-  if (order != nullptr) {
+  if (order) {
     for (OrderDescription* desc : *order) {
       delete desc;
     }
     delete order;
   }
 
-  if (withDescriptions != nullptr) {
+  if (withDescriptions) {
     for (WithDescription* desc : *withDescriptions) {
       delete desc;
     }
     delete withDescriptions;
   }
 
-  if (setOperations != nullptr) {
+  if (setOperations) {
     for (SetOperation* setOperation : *setOperations) {
       delete setOperation;
     }
     delete setOperations;
   }
 
-  if (lockings != nullptr) {
+  if (lockings) {
     for (LockingClause* lockingClause : *lockings) {
-      if (lockingClause->tables != nullptr) {
+      if (lockingClause->tables) {
         for (char* dtable : *lockingClause->tables) {
-          if (dtable != nullptr) free(dtable);
+          if (dtable) free(dtable);
```

Comment:
Simply `free(dtable)`?

Reply:
cppreference:
> If ptr is a null pointer, the function does nothing. 

Absolutely, gonna remove the mentioned ifs.